backend/api/annotations/resources.py

```diff
@@ -1,153 +1,169 @@
-        ProjectService.exists(project_id)
-        if annotation_type:
-            annotations = TaskAnnotation.get_task_annotations_by_project_id_type(
-                project_id, annotation_type
-            )
-        else:
-            annotations = TaskAnnotation.get_task_annotations_by_project_id(project_id)
-        return annotations.to_primitive(), 200
 
-    def post(self, project_id: int, annotation_type: str):
-        """
-        Store new task annotations for tasks of a project
-        ---
-        tags:
-            - annotations
-        produces:
-            - application/json
-        parameters:
-            - in: header
-              name: Content-Type
-              description: Content type for post body
-              required: true
-              type: string
-              default: application/json
-            - name: project_id
-              in: path
-              description: Unique project ID
-              required: true
-              type: integer
-            - name: annotation_type
-              in: path
-              description: Annotation type
-              required: true
-              type: string
-            - name: Application-Token
-              in: header
-              description: Application token registered with TM
-              required: true
-              type: string
-            - in: body
-              name: body
-              required: true
-              description: JSON object for creating draft project
-              schema:
-                projectId:
-                    type: integer
-                    required: true
-                annotationType:
-                    type: string
-                    required: true
-                tasks:
-                    type: array
-                    required: true
-                    items:
-                        schema:
-                            taskId:
-                                type: integer
-                                required: true
-                            annotationSource:
-                                type: string
-                            annotationMarkdown:
-                                type: string
-                            properties:
-                                description: JSON object with properties
-        responses:
-            200:
-                description: Project updated
-            400:
-                description: Client Error - Invalid Request
-            404:
-                description: Project or task not found
-            500:
-                description: Internal Server Error
-        """
+@router.get("/{project_id}/annotations/{annotation_type}/")
+@router.get("/{project_id}/annotations/")
+async def get(
+    request: Request,
+    project_id: int,
+    annotation_type: str = None,
+    user: AuthUserDTO = Depends(login_required),
+    db: Database = Depends(get_db),
+):
+    """
+    Get all task annotations for a project
+    ---
+    tags:
```

Comment:
Entirely optional, but you could always delete this docstrings. Fastapi generates the OpenAPI docs using typing anyway, so it's a bit redundant. It also means you need to remember to update the docstring if you update variables